recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,3 @@
-python:
-  - 3.8.* *_cpython
```

Comment:
3.8 isn't supported anymore, so this line probably has probably caused many problems.

Reply:
https://github.com/conda-forge/torchsde-feedstock/pull/8/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR12 Especially since the recipe told to skip 3.8